### ISSUE TITLE
inference code

### ISSUE THREAD

**Issue by tubixiansheng** [JCruan519/VM-UNet#53]
Hello, I'd like to ask if there are any inference codes available?Looking forward to your response.

**Comment by JCruan519** [comment]
Hello, the inference code and the training code are merged together. That is, after training, it will automatically select the best weights for inference testing. If you want to use the inference part separately, you can refer to the `test_one_epoch` function in `eggine.py`.

**Comment by hafsa390** [comment]
Hi @JCruan519 , looks like the validation dataset is used as test dataset during inference for isic.

**Comment by JCruan519** [comment]
@hafsa390 
Yes, we divide the training and test sets in a 7:3 ratio. This is consistent across all models.
